### PR TITLE
Add note about syncing to another Neo4j DB

### PR DESCRIPTION
**Pull request by l-heemann**
As per request: https://neo4j.slack.com/archives/C041SAJDE11/p1717406000194309?thread_ts=1717318241.168549&cid=C041SAJDE11

### CODE REVIEW

File: modules/ROOT/pages/get-started/self-managed.adoc [hunk 1]
```diff
@@ -142,3 +142,15 @@ In case large transactions resulted in log files larger than 256MB, the minimum
...
+
+=== Syncing to another Neo4j database
+CDC is a suitable tool for copying all or some data change events from one database to another, as well as building software that reacts to data change events in the database.
+
+However, it is not a suitable tool for creating an exact copy of your Neo4j database.
+Whilst the change event includes metadata about the transaction, it cannot be used to create identical elements in another Neo4j instance.
+Since the downstream database is a different instance, it creates its own IDs and timestamps when the data is transferred.
+The result is two databases that look similar, but have slight inconsistencies.
+
+Fault tolerance can be achieved by hosting Neo4j in a link:{neo4j-docs-base-uri}operations-manual/current/clustering/introduction/[cluster].
```
Comment (l-heemann) [inline modules/ROOT/pages/get-started/self-managed.adoc:154]: Note that this is omitted in the Aura page

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cdc-69.surge.sh

Updated pages:
- [CDC on Neo4j Aura](https://neo4j-docs-cdc-69.surge.sh/cdc/5/get-started/aura)
- [CDC on Neo4j DBMS](https://neo4j-docs-cdc-69.surge.sh/cdc/5/get-started/self-managed)

File: modules/ROOT/pages/get-started/aura.adoc [hunk 2]
```diff
@@ -67,3 +67,12 @@ Normally this is not an issue, as applications are meant to consume changes in (
...
 
+=== Syncing to another Neo4j database
+CDC is a suitable tool for copying all or some data change events from one database to another, as well as building software that reacts to data change events in the database.
+
+However, it is not a suitable tool for creating an exact copy of your Neo4j database.
+Whilst the change event includes metadata about the transaction, it cannot be used to create identical elements in another Neo4j instance.
+Since the downstream database is a different instance, it creates its own IDs and timestamps when the data is transferred.
+The result is two databases that look similar, but have slight inconsistencies.
+
+Offline copies of databases can be created using link:{neo4j-docs-base-uri}/operations-manual/current/backup-restore/[backups].
```
Comment (l-heemann) [inline modules/ROOT/pages/get-started/aura.adoc:78]: I'm not sure if we should include this. Could we point towards the backup and copy buttons in Aura? If so, how would we write something like that?

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cdc-69.surge.sh

Updated pages:
- [CDC on Neo4j Aura](https://neo4j-docs-cdc-69.surge.sh/cdc/5/get-started/aura)
- [CDC on Neo4j DBMS](https://neo4j-docs-cdc-69.surge.sh/cdc/5/get-started/self-managed)
- [Introduction](https://neo4j-docs-cdc-69.surge.sh/cdc/5)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cdc-69.surge.sh

Updated pages:
- [CDC on Neo4j Aura](https://neo4j-docs-cdc-69.surge.sh/cdc/5/get-started/aura)
- [CDC on Neo4j DBMS](https://neo4j-docs-cdc-69.surge.sh/cdc/5/get-started/self-managed)
- [Introduction](https://neo4j-docs-cdc-69.surge.sh/cdc/5)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cdc-69.surge.sh

Updated pages:
- [Introduction](https://neo4j-docs-cdc-69.surge.sh/cdc/5)

**Pull request merged**